### ISSUE TITLE
Cast & More actor items not working when inside an addon/browsing (MyVideoNav.xml)

### ISSUE THREAD

**Issue by inb4after** [SerpentDrago/skin.auramod#88]
Please view this video:

https://github.com/SerpentDrago/skin.auramod/issues/16#issuecomment-838742392

Open any addon such as TMDBHelper/Seren and browser the lists. Then Context menu > Information an item and then press Cast & More.

When selecting an actor, the window will close out due to TMDBHelper detecting changes when DialogVideoInfo.xml (PersonInfo) opens over the top of MyVideoNav.xml. 

Video items in Cast & More seem to work fine.

For now, we can still use homescreen widgets as the initial point of entry for "Cast & More" and the custom actor window seems to work as intended.

**Comment by adamosborne83** [comment]
Thank you 😊

**Comment by Dealz-416LS** [comment]
> 
> 
> > Its normal for it to flash back to last screen before opening up the New info page . then when you go back it will flash and load the last info page you were on .
> 
> Cheers for the reply, I guessed that, but if you see I clicked on the Girl, and the previous character I clicked on the bloke came up and then it crashed back to the hone screen as such and dis not show the girls info.
> Hope that makes sense

I was able to replicate, It seems that if you navigate to a cast members screen, then to a different cast member it will load the cast member that you looked at last, also as mentioned the navigation is automatically sent back to Netflix Info when Cast screen loads. 

For clarity I've added an additional video clip of the issue(s).

https://user-images.githubusercontent.com/54026944/117855485-70080b00-b282-11eb-8739-5c5e5f36169b.mp4

**Comment by adamosborne83** [comment]
I do just want to add, please don't feel all I'm doing is finding faults.
I wish I had the ability to do what you do, and I am ever so thankful for the effort you've all been doing to maintain and grow this.
I did do a shout out on Reddit MamouthBullfrog, awesome name I know.
So cheers again @inb4after @SerpentDrago

**Comment by inb4after** [comment]
> 
> 
> I do just want to add, please don't feel all I'm doing is finding faults.
> I wish I had the ability to do what you do, and I am ever so thankful for the effort you've all been doing to maintain and grow this.
> I did do a shout out on Reddit MamouthBullfrog, awesome name I know.
> So cheers again @inb4after @SerpentDrago

I literally would not have known about this, that's a good thing. At least it's semi-working in it's current state.

**Comment by SerpentDrago** [comment]
i mean it is called github "issues" not "fantastic things that work great" 

its all good , i'm sure 1000's of people use the skin daily that are fine for the most part and just ignore the issues without reporting.  . nothing wrong with being a tester and reporting the issues instead of just ignoring them . its what this issue section is for ! :)

**Comment by adamosborne83** [comment]
> > I do just want to add, please don't feel all I'm doing is finding faults.
> > I wish I had the ability to do what you do, and I am ever so thankful for the effort you've all been doing to maintain and grow this.
> > I did do a shout out on Reddit MamouthBullfrog, awesome name I know.
> > So cheers again @inb4after @SerpentDrago
> 
> I literally would not have known about this, that's a good thing. At least it's semi-working in it's current state.

I would NEVER use the Cast section tbf, I was just playing, you've git the movie aspect working and that's fab!!!

I'm just trying to break your hard work haha

**Comment by inb4after** [comment]
@adamosborne83 @Dealz-416LS 

Fixed in [Testing branch](https://github.com/SerpentDrago/skin.auramod/tree/Testing).

Please let us know your results, and if any new bugs occur.

**Comment by SerpentDrago** [comment]
Looks good here !

**Comment by Dealz-416LS** [comment]
Looks good here also!

last/previous cast member loading instead of selected is fixed.
Auto navigation back to Netflix Info when cast member screen loads is fixed.

**Comment by adamosborne83** [comment]
Sound as a pound here.....
I'll see what else I can try and break...... 

Must be repo time soon ;)

**Comment by adamosborne83** [comment]
Maybe one thing...... 
It kicks me out of the screen when I press return......
Anyone else had this? 

http://imgur.com/a/Hp0YV7A

**Comment by inb4after** [comment]
> Maybe one thing...... 
> It kicks me out of the screen when I press return......
> Anyone else had this? 
> 
> http://imgur.com/a/Hp0YV7A

Yeah, seems normal at the moment, still working on saving the initial info screen.

**Comment by adamosborne83** [comment]
That's fine, no major issue that it kicks me out the info screen, just playing and moaning as I do fella.

Have yourself an extra DrPepper tonight for the sugar rush

**Comment by adamosborne83** [comment]
Now, what I'm about as ask, might be an enhancement.... but say yes or no either way......

When you click a cast member, is there a way to have anything so you can see the movies they're in???

**Comment by SerpentDrago** [comment]
> Now, what I'm about as ask, might be an enhancement.... but say yes or no either way......
> 
> When you click a cast member, is there a way to have anything so you can see the movies they're in???

@adamosborne83
about to blow your mind
that already exists ! 

when you have a cast member info screen up just press down ! lol  (there is a little indicator that you can do this ) 


https://user-images.githubusercontent.com/21133858/118021676-7d2c0500-b329-11eb-9fa6-3695533189d2.mp4

**Comment by adamosborne83** [comment]
> > Now, what I'm about as ask, might be an enhancement.... but say yes or no either way......
> > When you click a cast member, is there a way to have anything so you can see the movies they're in???
> 
> @adamosborne83
> about to blow your mind
> that already exists !
> 
> when you have a cast member info screen up just press down ! lol (there is a little indicator that you can do this )
> 
>  Kodi.2021-05-12.13-53-15.mp4

Shush, who bloody asked you anyway haha.

As soon as I'm out the hot tub I'll check haha

**Comment by adamosborne83** [comment]
OK, so please try a make it to the end of the video.

It's very odd, tbf feel free to say shut up Adam, but just proves I'm not totally crazy.

For a little reference, I'm using a firecube, with a totally clean Kodi 19, seren for main, Venom as back up, open wizard, a4k subtitles and ofcpurse the amazing auramod skin.
So a real simple set up.

Enjoy the video

http://imgur.com/a/1gzhwxW

http://imgur.com/a/VTlLEev

For some reason would do the whole video I saved. So in 2 parts

@SerpentDrago

**Comment by Dealz-416LS** [comment]
Looks like the ability to press down on the cast member screen is lost as the screen is loading.

If you watch the video as the cast member screen is loading you can see the two down arrows (indicator) appear but then they disappear just before the screen fully loads resulting in pressing down not functioning.

Bringing up the context menu does allow access to the "Cast in Movies" option among others but non are functioning.

![image](https://user-images.githubusercontent.com/54026944/118031704-be8ad780-b35e-11eb-98f5-dd83dda8f4e4.jpeg)

**Comment by adamosborne83** [comment]
> Looks like the ability to press down on the cast member screen is lost as the screen is loading.
> 
> If you watch the video as the cast member screen is loading you can see the two down arrows (indicator) appear but then they disappear just before the screen fully loads resulting in pressing down not functioning.
> 
> Bringing up the context menu does allow access to the "Cast in Movies" option among others but non are functioning.
> 
> ![image](https://user-images.githubusercontent.com/54026944/118031704-be8ad780-b35e-11eb-98f5-dd83dda8f4e4.jpeg)

Yea I did notice that too, but found it odd that in the second video it allowed me too......
I didnalso try the context menu, but when I clicked  Cast in Movies it did nothing. Maybe its a limitation of amazon cube

**Comment by SerpentDrago** [comment]
what ? i've never once not been able to go down on a actor . you will need to replicate this . 

also this is all getting extremely off topic , if you have anymore issues please open a new issue post ! ;0

**Comment by SerpentDrago** [comment]
> OK, so please try a make it to the end of the video.
> 
> It's very odd, tbf feel free to say shut up Adam, but just proves I'm not totally crazy.
> 
> For a little reference, I'm using a firecube, with a totally clean Kodi 19, seren for main, Venom as back up, open wizard, a4k subtitles and ofcpurse the amazing auramod skin.
> So a real simple set up.
> 
> Enjoy the video
> 
> http://imgur.com/a/1gzhwxW
> 
> http://imgur.com/a/VTlLEev
> 
> For some reason would do the whole video I saved. So in 2 parts
> 
> @SerpentDrago

please make another issue post , and describe whats going on not just a video ;0 videos and pictures are great but a description of what your doing is also nice lol . what addon your in , etc etc . anything needed to replicate the issue . other wise we have to guess and try a million things . 

also you know you can just upload movies to github right ? just drag and drop them in the comment

**Comment by SerpentDrago** [comment]
@adamosborne83 your missing a bunch of menus for cast and more . your prob being rate limited look at your logs . which is causing sections and lookups to fail . 

stop testing so hard lol

**Comment by inb4after** [comment]
> 
> 
> Maybe one thing......
> It kicks me out of the screen when I press return......
> Anyone else had this?
> 
> http://imgur.com/a/Hp0YV7A

This secondary issue has been fixed in [Testing branch](https://github.com/SerpentDrago/skin.auramod/tree/Testing).

**Comment by adamosborne83** [comment]
> > Maybe one thing......
> > It kicks me out of the screen when I press return......
> > Anyone else had this?
> > http://imgur.com/a/Hp0YV7A
> 
> This secondary issue has been fixed in [Testing branch](https://github.com/SerpentDrago/skin.auramod/tree/Testing).

Brilliant thank you.

**Comment by adamosborne83** [comment]
> @adamosborne83 your missing a bunch of menus for cast and more . your prob being rate limited look at your logs . which is causing sections and lookups to fail .
> 
> stop testing so hard lol

Sorry hahaha 😅 I consider myself told off 😅

**Comment by adamosborne83** [comment]
> > OK, so please try a make it to the end of the video.
> > It's very odd, tbf feel free to say shut up Adam, but just proves I'm not totally crazy.
> > For a little reference, I'm using a firecube, with a totally clean Kodi 19, seren for main, Venom as back up, open wizard, a4k subtitles and ofcpurse the amazing auramod skin.
> > So a real simple set up.
> > Enjoy the video
> > http://imgur.com/a/1gzhwxW
> > http://imgur.com/a/VTlLEev
> > For some reason would do the whole video I saved. So in 2 parts
> > @SerpentDrago
> 
> please make another issue post , and describe whats going on not just a video ;0 videos and pictures are great but a description of what your doing is also nice lol . what addon your in , etc etc . anything needed to replicate the issue . other wise we have to guess and try a million things .
> 
> also you know you can just upload movies to github right ? just drag and drop them in the comment

I'm using my phone on github, don't think I've got my computer out in years.

**Comment by Dealz-416LS** [comment]
> Yea I did notice that too, but found it odd that in the second video it allowed me too......
> 

@adamosborne83 

Is this working consistently for you now? 

Being able to press down on the cast member screen.

It's not working at all for me.

**Comment by adamosborne83** [comment]
> > Yea I did notice that too, but found it odd that in the second video it allowed me too......
> 
> @adamosborne83
> 
> Is this working consistently for you now?
> 
> Being able to press down on the cast member screen.
> 
> It's not working at all for me.

Nope not working, but can live without it tbf

**Comment by xyzfre** [comment]
https://user-images.githubusercontent.com/49699869/118204894-43442700-b42d-11eb-9e5f-3868766938a2.MOV



Same issue here pushing down after clicking on the actor doesn’t bring up no movies or shows on that actor using Nvidia shield K19.1 ,,Aura Home Layout , Vertical Icon Only ,, tried with Venom and Fen,, I tried with the testing branch and the matrix branch

**Comment by SerpentDrago** [comment]
ok are all you guys on android ? on pc and i've never had this issue , we need more info . could we get logs ?

**Comment by adamosborne83** [comment]
> ok are all you guys on android ? on pc and i've never had this issue , we need more info . could we get logs ?

Hey Mr,
I'm more than happy to open I new issue thread on this, now, this might be my ignorance, but would there be an issue log as it seems to just be an android box issue

**Comment by inb4after** [comment]
@adamosborne83 @Dealz-416LS @xyzfre 

I've tested on Shield and still cannot reproduce. It's a very strange issue that may be related to a certain setting being enabled/disabled, but I'm planning on making a few hotfixes for you all to try out.

I've opened a new issue #92 as it is separate to this one, I'll close this issue once Testing branch is merged into Matrix branch.

**Comment by inb4after** [comment]
Fixed in AuraMOD [Matrix branch](https://github.com/SerpentDrago/skin.auramod) v1.0.6
